sherlock_project/resources/data.json

```
@@ -1541,6 +1581,13 @@
     "urlMain": "https://www.patreon.com/",
     "username_claimed": "blue"
   },
+  "PayPal.me": {
+    "errorMsg": "We can't find this profile",
```

```suggestion
    "errorMsg": "We can’t find this profile",
```
Character mismatch `’` vs `'` leads to F+